hal/include/hal/ArduinoUnoAliases.h

```diff
@@ -0,0 +1,97 @@
+
+/** \addtogroup hal */
+/** @{*/
+/* mbed Microcontroller Library
+ * Copyright (c) 2020 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_ARDUINO_UNO_H
+#define MBED_ARDUINO_UNO_H
+
+#if defined (TARGET_FF_ARDUINO) || (TARGET_FF_ARDUINO_UNO)
+
+#ifdef TARGET_FF_ARDUINO_UNO
+
+#define A0  ARDUINO_UNO_A0
```

Comment:
Do we want to define `Ax` / `Dx`?
Perhaps only if `TARGET_FF_ARDUINO` is also defined?

Reply:
Goal is to keep compatibility with all libs using Ax and Dx